### PR TITLE
Added brief technical details on bottom

### PR DESCRIPTION
**Pull request by Gingeropolous**

### CODE REVIEW

File: knowledge-base/about.md [hunk 1]
```diff
@@ -63,3 +63,8 @@ Another issue with most cryptocurrencies is the development of a fee market. Thi
...
 Monero will enforce a minimum mixing across the network, so all outputs are mixed by default. This is possible due to the nature of the mixing: monero mixing is "passive" and can even be done offline! Transaction outputs have "plausible deniability" about their state: you can't tell if they are spent or unspent in a certain transaction or not. This leads to an opaque (non-transparent)  blockchain making all coins "equal". Fungibility is built into Monero at protocol level, making it real "digital cash". 
+
+## TECHNICAL DETAILS
+Proof of work = Cryptonite
```
Comment (dEBRUYNE-1) [inline knowledge-base/about.md]: I think you meant CryptoNight here?

**Comment by dEBRUYNE-1** [comment]
Left a comment.

**Comment by anonimal** [comment]
@Gingeropolous did you still want to merge this?

**Comment by Gingeropolous** [comment]
sure?

**Comment by anonimal** [comment]
@Gingeropolous I was simply asking because it's from Oct 29, 2016 and it needs to be rebased before merging.

**Comment by Gingeropolous** [comment]
Yeah I have no idea how to do that. I think I made the edits in github via
the github GUI.... not with the actual CLI git.

**Comment by fluffypony** [comment]
Closed, that way you can make the edits again via GitHub and resubmit:)

**Pull request closed without merging**